### ISSUE TITLE
Build and Classify with a Neural Network

### ISSUE THREAD

**Issue by triptithawait** [triptithawait/skills-copilot-codespaces-vscode#5]

**Comment by triptithawait** [comment]
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

def build_model(hidden_layers=[128, 64], use_bias=True):
    model = Sequential()
    model.add(Dense(hidden_layers[0], input_shape=(784,), activation='relu', use_bias=use_bias))
    for units in hidden_layers[1:]:
        model.add(Dense(units, activation='relu', use_bias=use_bias))
    model.add(Dense(10, activation='softmax', use_bias=use_bias))
    model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy'])
    return model
